examples/runDecamTest.sh

```diff
@@ -52,7 +52,7 @@ if [[ $DOPROCESS == true ]]; then
         -i ingestImagesDecam.py
 fi
 
-if [[ $DOVERIFY = true ]]; then
+if [[ $DOVERIFY == true ]]; then
```

Comment:
Interesting.  How did this ever work in the past?

Reply:
Apparently there's no difference between [[$x=true]] and [[$x==true]] in bash.